src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1539,6 +1549,68 @@ static bool removeEntity
 
 /* ****************************************************************************
 *
+* searchContextProviders -
+*
+*/
+void searchContextProviders(const std::string&              tenant,
+                            const std::vector<std::string>& servicePathV,
+                            EntityId&                       en,
+                            ContextAttributeVector&         caV,
+                            ContextElementResponse*         cerP)
+{
+
+  ContextRegistrationResponseVector  crrV;
+  EntityIdVector                     enV;
+  AttributeList                      attrL;
+  std::string                        err;
+
+  /* Fill input data for registrationsQuery() */
+  enV.push_back(&en);
+  for (unsigned int ix = 0; ix < caV.size(); ++ix)
+  {
+    attrL.push_back(caV.get(ix)->name);
+  }
+
+  /* First CPr lookup (in the case some CER is not found): looking in E-A registrations */
+  if (someContextElementNotFound(*cerP))
```

Comment:
Not exaclty... function signature is:

```
extern bool someContextElementNotFound(ContextElementResponse& cer);
```

If I ty to use `const ContextElementResponse& cer` I get the following compling error:

```
/home/fermin/src/fiware-orion/src/lib/mongoBackend/MongoGlobal.cpp: In function ‘bool someContextElementNotFound(const ContextElementResponse&)’:
/home/fermin/src/fiware-orion/src/lib/mongoBackend/MongoGlobal.cpp:2097:81: error: passing ‘const ContextAttributeVector’ as ‘this’ argument of ‘unsigned int ContextAttributeVector::size()’ discards qualifiers [-fpermissive]
/home/fermin/src/fiware-orion/src/lib/mongoBackend/MongoGlobal.cpp:2099:54: error: passing ‘const ContextAttributeVector’ as ‘this’ argument of ‘ContextAttribute* ContextAttributeVector::operator[](unsigned int)’ discards qualifiers [-fpermissive]
make[3]: *** [src/lib/mongoBackend/CMakeFiles/mongoBackend.dir/MongoGlobal.cpp.o] Error 1
make[3]: se sale del directorio `/home/fermin/src/fiware-orion/BUILD_UNITTEST'
make[2]: *** [src/lib/mongoBackend/CMakeFiles/mongoBackend.dir/all] Error 2
```

Not sure what could be happending, as "conceptually" is an object that I don't want to change. Any idea? (it would be great to solve this, as we have the same "problem" in may function signatures).